client/styles/components/_account.scss

```diff
@@ -10,6 +10,7 @@
   max-width: #{700 / $base-font-size}rem;
   align-self: center;
   padding: 0 #{10 / $base-font-size}rem;
+  padding: 10px;
```

Comment:
You'll want to update the previous line rather than adding a new line which overwrites it.

Reply:
Oh yes, I want to add padding-bottom because there is no need to add it at the top, so I will write the next line